src/System.Net.Http.WinHttpHandler/src/System/Net/Http/WinHttpHandler.cs

```diff
@@ -1661,8 +1661,8 @@ private void SetRequestHandleClientCertificateOptions(SafeWinHttpHandle requestH
             }
 
             // Reduce the set of certificates to match the proper 'Client Authentication' criteria.
-            certs = certs.Find(X509FindType.FindByKeyUsage, X509KeyUsageFlags.DigitalSignature, true);
-            certs = certs.Find(X509FindType.FindByApplicationPolicy, ClientAuthenticationOID, true);
+            certs = certs.Find(X509FindType.FindByKeyUsage, X509KeyUsageFlags.DigitalSignature, false);
+            certs = certs.Find(X509FindType.FindByApplicationPolicy, ClientAuthenticationOID, false);
```

Comment:
@bartonjs Thanks for the info on the particulars of how .Find() works.  I will incorporate this knowledge as part of my next PR for perf.